kong/db/schema/others/declarative_config.lua

```diff
@@ -830,6 +838,20 @@ local function flatten(self, input)
         end
       end
 
+      if schema.ttl then
+        local ttl_value = entry.ttl
+        if ttl_value and ttl_value ~= 0 and ttl_value ~= null then
+          if entry.updated_at then
+              ttl_value = ttl_value + entry.updated_at
+          elseif entry.created_at then
```

Comment:
should have a blank line before `elesif`, it is kong's style.

See: https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#conditional-expressions

Reply:
done

Reply:
blank line needed, should be 
```
          if entry.updated_at then
              ttl_value = ttl_value + entry.updated_at

          elseif entry.created_at then
              ...
          
          else
```

Reply:
From [doc](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#conditional-expressions): 
> For one-line blocks, blank lines are not necessary:
>
> --- good
> if foo then
>   do_stuff()
> else
>   error("failed!")
> end

HI @chronolaw , one-line block may not need blank